packages/backend-plugin-api/src/services/definitions/PermissionsService.ts

```diff
@@ -59,6 +61,16 @@ export interface PermissionsService extends PermissionEvaluator {
     options: PermissionsServiceRequestOptions,
   ): Promise<AuthorizePermissionResponse[]>;
 
+  /**
+   * Evaluates batch of
+   * {@link backstage/plugin-permission-common#Permission | Permissions} and
+   * returns a definitive decision for each.
+   */
+  authorizeBatch(
```

Comment:
doesn't the `authorize` method right above 👆do this already? 🤔

Reply:
Issue is that the `PermissionApi` from frontend doesn't do this and thus if you have to check multiple permissions it will require multiple HTTP requests.